modules/S21_python_check.sh

```diff
@@ -77,6 +77,9 @@ s21_script_bandit() {
   local NAME=""
   local PY_LOG=""
   local VULNS=""
+  local GPT_PRIO=3
+  local GPT_ANCHOR=""
+  local GPT_QUESTION_="Please identify all vulnerabilities in this python code and give me a describtion on how to exploit them:"
```

Comment:
could we use the same question everywhere and define only the language in the module?

Somewhere in the global variable area
```
export GPT_QUESTION="Please identify all vulnerabilities in this ${LANGUAGE} code and give me a describtion on how to exploit them:"
```

and here we can do
local LANGUAGE="python"

Does this work?